xr_sample_project/Assets/SampleViewer/Samples/VRSample/Scripts/SetTurnType.cs

```diff
@@ -9,6 +9,9 @@ public class SetTurnType : MonoBehaviour
     [SerializeField] private ActionBasedContinuousTurnProvider continuousTurn;
     [SerializeField] private ActionBasedSnapTurnProvider snapTurn;
 
+    public delegate void ChangedTurnType(bool type);
+    public event ChangedTurnType OnTypeChanged;
+
     public void SetTurnTypeFromIndex(int index)
```

Comment:
Why do we need to set it via index? I feel like this could be avoided wherever this is being called.